colearn_grpc/grpc_server.py

```diff
@@ -56,12 +57,37 @@ def run(self):
 
         _logger.info(f"Starting GRPC server on {address}...")
 
+        # There needs to be a certificate and private key available to enable encryption -
+        # if these are not available, fall back to no encryption.
+        encrypted_connection = True
+
+        if not os.path.isfile("server.crt"):
+            _logger.error("Failed to find file server.crt needed for encrypted grpc connection - not enabling")
+            encrypted_connection = False
+
+        if not os.path.isfile("server.key"):
```

Comment:
These filenames are used twice and could be in constants. Also a path that is relative to the repo root would be better!

Reply:
E.g.
```python
REPO_ROOT = Path(__file__).absolute().parent.parent
server_key = REPO_ROOT / "server.key"
```

Reply:
Not twice? server.crt and server.key. But you are right, it would be cleaner to have them as constants etc.